samples/tests/mission1-local.yaml

```diff
@@ -6,8 +6,17 @@ drone:
 simulation:
   simulator: ros #{gazebo, jmavsim, ros} 
   speed: 1
-  headless: false
-  obstacles: [5,5,5,  -8,3,0,  0] # size,position,orientation of a box
+  headless: true
+  obstacles:
+  - size:
+      l: 6
```

Comment:
For consistency with other sample tests, let's keep the size and positions as before

Reply:
Done